src/rpc/misc.cpp

```diff
@@ -1077,8 +1077,8 @@ UniValue getspentinfo(const UniValue& params, bool fHelp)
             "  ,...\n"
             "}\n"
             "\nExamples:\n"
-            + HelpExampleCli("getspentinfo", "'{\"txid\": \"0437cd7f8525ceed2324359c2d0ba26006d92d856a9c20fa0241106ee5a597c9\", \"index\": 0}'")
-            + HelpExampleRpc("getspentinfo", "{\"txid\": \"0437cd7f8525ceed2324359c2d0ba26006d92d856a9c20fa0241106ee5a597c9\", \"index\": 0}")
+            + HelpExampleCli("getspentinfo", "'{\"txid\": \"33990288fb116981260be1de10b8c764f997674545ab14f9240f00346333b780\", \"index\": 4}'")
+            + HelpExampleRpc("getspentinfo", "{\"txid\": \"33990288fb116981260be1de10b8c764f997674545ab14f9240f00346333b780\", \"index\": 4}")
```

Comment:
Why are these altered?

Reply:
That's so they match the documentation here: file:///home/larry/gitlab/zcash-docs/build/html/rtd_pages/insight_explorer.html (which hasn't been merged yet), and the reason for those values there is that all the examples actually work on testnet. (I think the previous value, `0437cd...` is from bitcoin.)

Reply:
yes, it's from bitcoin mainnet block 9: https://blockchair.com/bitcoin/transaction/0437cd7f8525ceed2324359c2d0ba26006d92d856a9c20fa0241106ee5a597c9